doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -913,6 +923,22 @@ Note that if a custom payload is used for the notification (the field `payload`
 corresponding subscription), then a value of `custom` is used for the `Ngsiv2-AttrsFormat` header
 in the notification.
 
+## Special metadata in notifications
+
+The following metadata are not included in notifications by default, the `metadata` field (within
+`notification` field) in subscription is used to do so.
+
+* `previousValue`: the value and type of this metadata is the one of the attribute to which it
+  is attached previous to the processing of the request that triggered the notification. If the
+  type/value of `previousValue` is the same type/value than in the attribute in which it has been
+  included, then the attribute has not actually changed its value.
+
+* `actionType`: it is included if the attribute to which it is attached was included in the request
+  that triggered the notification. Its value depend on the request operation type: `update`
+  for updates, `append` for creation and `delete` for deletion. Its type is always `Text`.
+  (NOTE: currently Orion, reference implementation of NGSIv2 API, doesn't implement the append
+  and delete cases).
```

Comment:
Note moved to the ngsiv2_implementation_notes.md file. Fixed in 9a645cd8244a7c4023dc0eaaee610c87c3f351df.